clang/test/CIR/CodeGen/fun-ptr.c

```diff
@@ -0,0 +1,44 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o -  | FileCheck %s -check-prefix=CIR
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-llvm %s -o - | FileCheck %s -check-prefix=LLVM
```

Comment:
Great, can you also add RUN lines that build this file in c++ mode? E.g: `-x cpp -std=c++20` should do it.